test/e2e/missing-module/test.js

```diff
@@ -3,6 +3,7 @@
 
 const path = require('path');
 const testEnvs = require('../errors.js');
+const t = require('assert');
```

Comment:
(Minor) linter is complaining that this is never used:

{code}
$ eslint .
--
  | Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
  | { parser: "babylon" } is deprecated; we now treat it as { parser: "babel" }.
  |  
  | /fusion-cli/test/e2e/missing-module/test.js
  | 6:7  error  't' is assigned a value but never used  no-unused-vars
  |  
  | ✖ 1 problem (1 error, 0 warnings)


{code}